app/mobile/lib/view/resource/resourcepage.dart

```diff
@@ -42,16 +49,70 @@ Widget resourcePageView(TopicDetailed t, Resource r, ChangeNotifier topicPageVie
                               }
                             },
                             styleSheet: MarkdownStyleSheet(
-                                blockquoteDecoration:
-                                    const BoxDecoration(color: CustomColors.main)),
+                                blockquoteDecoration: const BoxDecoration(
+                                    color: CustomColors.main)),
                             data: r.body,
                             shrinkWrap: true,
                             physics: const ClampingScrollPhysics(),
                           ),
                         ],
                       ),
                     ),
-                    editResourceButton(t, r, context, topicPageView),
+                    Padding(
+                      padding: const EdgeInsets.all(10.0),
+                      child: AnimatedFloatingActionButton(
+                          fabButtons: <Widget>[
+                            FloatingActionButton(
```

Comment:
We should show the edit button only if the logged in user is the creator of the resource.

Reply:
I fixed this part please check